recipes/smncopynumbercaller/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin
+chmod 755 smn_caller.py
+cp -r ./* $PREFIX/bin/
```

Comment:
This will copy all of the files from the repo checkout to the bin directory, you should select based on extension, e.g. *.py.

Reply:
Actually, the conventional way for handling Python stuff is to make a pypi package first, and then wrap it in Bioconda using the pypi skeleton, or Grayskull etc.

Reply:
Thanks, I will do that.